erdos/src/dataflow/stream/ingress_stream.rs

```diff
@@ -0,0 +1,159 @@
+use std::{
+    sync::{Arc, Mutex},
+    thread,
+    time::Duration,
+};
+
```

Comment:
Can you git rename this file instead of deleting and restarting it? It would be nice to have commit history.